assets/templateFloxEnv/manifest.toml

```diff
@@ -20,13 +20,21 @@
 # pass-in = "$some-env-var" 
 
 # An activation hook will be run when entering the environment.
-# You can define one in 'hook' table with an inline 'script' field,
+# You can define one in the 'hook' table inline via the 'script' field,
 # or provide a file using the 'file' field.
 # If 'file' is provided, the path should be relative to this file.
 # If both 'file' and 'script' are provided,
-# only the hook defined in 'file' will be run 
+# only the hook defined in 'file' will be run .
 
 [hook]
 # script = """
 #   echo "it's gettin flox in here";
 # """
+
+# An environment that works on one system is guaranteed to work on the
+# same type of system,
+# but other systems may not have the same packages available, etc.
+# In order to use the environment on a system you must explicitly
+# add it to this list.
+[options]
+systems = ["_FLOX_INIT_SYSTEM"]
```

Comment:
Can't we put an empty list here and fill that with toml_edit?
The options table or even the options.systems should be enough of an anchor no?
Using m4 seems like a weird choice here (here, m4 may have its place in the world)

Reply:
That's an option, but it actually turns out to be much simpler to just use `m4`. With `toml_edit` you also have to read "options", make sure it's a table, read "systems", make sure it's an array, etc. Much simpler to just say "replace this string with that one."

Reply:
Sure it's simpler, a bit fragile but okk.


But if you just want to replace a string... why don't you just [`replace` a string](https://doc.rust-lang.org/std/primitive.str.html#method.replace). For just substituting a known string you don't need m4?

Reply:
It's not fragile if we own the template and have a test. I'm replacing the string now.